### DISCUSSION TITLE
BUG: new projects do not allow task assignments to external collaborators

### DISCUSSION THREAD

**Discussion by securestep9** [community/community#34091] [Projects and Issues]
I migrated a Classic project into a new project and spotted that there is a bug. If I go to project settings/Manage access I can see 12 external collaborators who have write access to the project. However when I open a task and try to assign it none of these 12 project collaborators are visible, for the assignment

**Comment by ossanna16** [comment]
Thank you for reporting this bug @securestep9! I'm glad you found our community!
